charts/trino/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.8.0
+version: 0.9.0
```

Comment:
reminder to self: release 0.9.0 with other pending PRs and unreleased changes and then update this to 0.10.0.